src/db/spanner/models.rs

```diff
@@ -647,12 +650,21 @@ impl SpannerDb {
         }
     }
 
+    fn erect_tombstone(&self, user_id: u32) -> Result<()> {
+        self.sql("INSERT INTO user_collections (userid, collection, last_modified) values (@userid, \"0\", @modified)")?
+            .params(params!{"userid" => user_id.to_string(), "modified" => self.timestamp().as_rfc3339()?})
```

Comment:
need an explicit type for timestamps:
```suggestion
            .params(params!{"userid" => user_id.to_string(), "modified" => self.timestamp().as_rfc3339()?})
            .param_types(param_types! {
                    "modified" => SpannerType::Timestamp,
                })            
```